buildenv/jenkins/common/test

```diff
@@ -113,20 +113,20 @@ def test_all() {
         compile()
         test()
         publish()
+
+        try {
+            cleanWs()
+        } catch(e) {
+            echo e.toString()
+            if (SPEC.startsWith("win_")) {
+                sh "ps -W"
```

Comment:
Can you add a comment to the file describing the windows special case?

Reply:
https://github.com/eclipse/openj9/commit/ed9f4a28d000459e23a072ddc943cb2c17c58a10
I think we were having some cases where the `cleanWs()` would fail on Windows. I believe this was an attempt to help debug what may be holding the file(s) open when it failed.

Reply:
@AdamBrousseau please add that comment into the code

Reply:
I'm not sure why we would limit it to only run on Windows specs. Pushing a new commit with the comment.